src/seedu/addressbook/data/person/Address.java

```diff
@@ -55,4 +57,17 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    @Override
+    public String getPrintableString() {
+        StringBuilder builder = new StringBuilder();
+        builder.append(LABEL);
+        
+        if (isPrivate()) {
+            builder.append(LABEL_PRIVATE);
```

Comment:
I think the exercise said to append "LABEL: value" :) You have gone one step ahead.